blackbox/docs/config/logging.rst

```diff
@@ -99,3 +99,42 @@ Settings<sys-cluster-settings>`).
 
 .. _Log4j: https://logging.apache.org/log4j/1.2/
 .. _`hierarchical named loggers`: https://logging.apache.org/log4j/1.2/manual.html
+
+.. _conf-logging-gc:
+
+Garbage Collection Logging
+==========================
+
+Addition to the regular application logging, CrateDB also logs garbage
+collection times of the Java Virtual Machine using the built-in GC logging of
+the JVM.
+
+GC logging is enabled by default since CrateDB 3.0. It can be disabled by
+setting the environment variable ``CRATE_DISABLE_GC_LOGGING``.
+
+The default location for the GC log files is ``$CRATE_HOME/logs``. The files
+are logrotated at 64MB in size, kept 16 rounds, and prefixed with ``gc.log``.
+This means, that the GC log files may take up to 1GB of space on your disk.
+However, both the maximum file size and the amount of kept files can be
+controlled with environment variables.
```

Comment:
link to more info about env vars? or provide them?

Reply:
initially I described the env vars both in prosa and in short (below). but I think the overview below is enough. I can add a ref to it